app/views/apply/new-eligibility/about-the-prisoner.html

```diff
@@ -110,13 +110,28 @@ <h1 class="govuk-heading-l">
         errorMessage: errors | findError('PrisonerNumber')
       }) }}
       {% else %}
-        <label id="PrisonerNumber" class="govuk-label" for="prisoner-number">Prisoner number</label>
+        <label id="PrisonerNumber" class="govuk-label" for="prisoner-number">Prison number</label>
         <span>{{ prisonerNumber }}</span>
         <input id="prisoner-number" name="PrisonerNumber" value="{{ prisonerNumber }}" type="hidden">
       {% endif %}
 
+      <details class="govuk-details">
+        <summary class="govuk-details__summary">
+          <span class="govuk-details__summary-text">
+            Help with the prison number
+          </span>
+        </summary>
+        <div class="govuk-details__text">
+          This is the number given to people when they enter prison.
```

Comment:
For niceness this should be using `<p>` tags, `<br>` are not for formatting but won't decline because of it 👍